sql-schema/256.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE ports
+  MODIFY `ifInUcastPkts` BIGINT UNSIGNED,
```

Comment:
each command must be a single line, so either collapse to one line, or make multiple alter statements.

Reply:
Changed to multiple alter statements